deployment/src/main/java/io/quarkiverse/dapr/deployment/DaprProcessor.java

```diff
@@ -174,6 +179,17 @@ void addTopic(DaprRuntimeRecorder daprRuntimeRecorder, List<DaprTopicBuildItem>
                     item.getPriority(),
                     item.getRoute(),
                     item.getMetadata());
+
+            if (!item.getRoute().isBlank()) {
+                produceReflectiveClasses = true;
+            }
+        }
+
+        if (produceReflectiveClasses) {
+            reflectiveClasses.produce(ReflectiveClassBuildItem.builder(
+                    DaprTopicRule.class.getName(),
+                    DaprTopicRoutes.class.getName(),
+                    CloudEventReader.class.getName()).build());
```

Comment:
Just wonder why these classes need to be registered for reflection only in route is not blank if I read the codes correctly?

And it seeem that `io.quarkiverse.dapr.core.DaprTopicSubscription` is missed here?

Reply:
>Just wonder why these classes need to be registered for reflection only in route is not blank if I read the codes correctly?

I added this because I saw that in the last PR it was necessary to add annotation to the client code after using @Route.
Can I register it even if it is not used in the native image? Is graalvm smart enough not to use the reflection statement? 

> And it seeem that io.quarkiverse.dapr.core.DaprTopicSubscription is missed here?

This one is produced by `void reflectiveClasses(BuildProducer<ReflectiveClassBuildItem> reflectiveClassBuildProducer)` method.

Reply:
OK, in this case, it should be OK to add all of them.